src/App.tsx

```diff
@@ -14,11 +15,95 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string[],
+};
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGood: ['Jam'],
+  };
 
+  addGood = (good:string) => {
```

Comment:
you can delete good on this method 

```suggestion
  updateGoods = (good:string) => {
```